wolfhsm/wh_message_nvm.h

```diff
@@ -47,9 +48,8 @@ enum WH_MESSAGE_NVM_ACTION_ENUM {
 
 enum WH_MESSAGE_NVM_MAX_ENUM {
     /* must be odd for struct whMessageNvm_DestroyObjectsRequest  alignment */
-    WH_MESSAGE_NVM_MAX_DESTROY_OBJECTS_COUNT = 9,
-    WH_MESSAGE_NVM_MAX_ADD_OBJECT_LEN =
-            WH_COMM_DATA_LEN - sizeof(whNvmMetadata),
+    WH_MESSAGE_NVM_MAX_DESTROY_OBJECTS_COUNT = 19,
```

Comment:
change from 9->19?

Reply:
If the MTU has to be no smaller than about 200 bytes, there's really no reason not to expand this a bit.